tests/FileStorageS3Test.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+
+namespace Keboola\ManageApiTest;
```

Comment:
Testy po sobě neuklízí ty filestorage! 

![image](https://user-images.githubusercontent.com/642928/60343961-c2edb780-99b5-11e9-8563-79bbaf864c63.png)

Reply:
Jo to je wanted. Je to omezený env proměnnou, aby se tyhle testy zatím vůbec nepouštěly. Už jsem to psal někde výš, ale ono to mazaní nejde, protože by to smazalo všechny filestorage a ty by si neměl kam zapisovat. Jsou to jenom vývojový testy (stejně jako byly před tím)